.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
 site
+run_dateentry.py
```

Comment:
Dej do .git/info/exclude – to je soukromé pro tebe, .gitignore tady nepoužij (protože se nečeká, že by ten soubor měl někdo další)